presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/DesugarAtTimeZone.java

```diff
@@ -30,29 +29,25 @@
 import java.util.Set;
 
 public class DesugarAtTimeZone
-        implements RuleSet
 {
     private final Metadata metadata;
     private final SqlParser sqlParser;
 
-    private Set<Rule<?>> rules = ImmutableSet.of(
-            new ProjectExpressionRewrite(this::rewrite),
-            new AggregationExpressionRewrite(this::rewrite),
-            new FilterExpressionRewrite(this::rewrite),
-            new TableScanExpressionRewrite(this::rewrite),
-            new JoinExpressionRewrite(this::rewrite),
-            new ValuesExpressionRewrite(this::rewrite));
```

Comment:
This is unreleated

Reply:
I inlined field into `rules()` method, since there is no gain in having it.